UT4MasterServer/Exceptions/InvalidEpicIDException.cs

```diff
@@ -5,7 +5,47 @@ namespace UT4MasterServer.Exceptions;
 [Serializable]
 public class InvalidEpicIDException : Exception
 {
-	public InvalidEpicIDException(string message) : base(message) { }
+	private string _id { get; set; }
+	public string ID { get { return _id; } }
 
-	protected InvalidEpicIDException(SerializationInfo serializationInfo, StreamingContext streamingContext) : base(serializationInfo, streamingContext) { }
+	private int _numericErrorCode { get; set; }
+	public int NumericErrorCode { get { return _numericErrorCode; } }
+
+	private string _errorCode { get; set; }
+	public string ErrorCode { get { return _errorCode; } }
```

Comment:
Is there a reason for having a set of private/public properties? public property could just have setter marked as private